src/tasks/proxy/list.js

```diff
@@ -0,0 +1,77 @@
+const { Logger } = require('KegLog')
+const { getProxyRoutes } = require('KegUtils/proxy/getProxyRoutes')
+const { filterProxyRoutes } = require('KegUtils/proxy/filterProxyRoutes')
+const { noOpObj, wordCaps, mapObj } = require('@keg-hub/jsutils')
+
+/**
+ * Logs the items returned from the traefik api to the terminal
+ * @param {Array} items - Items returned from traefik api after being filtered
+ *
+ * @returns {Void}
+ */
+const logList = list => {
+  Logger.subHeader(`Keg-Proxy Container Routes`)
+  const groups = {}
+  // Convert the array into an object grouped by app name
+  list.map(item => {
+    const [ name, ...rest ] = item.service.split('-')
+    groups[name] = groups[name] || []
+    groups[name].push({
+      branch: `    * ${rest.join('-')}`,
+      url: `    * https://${item.rule.split('`')[1]}`
+    })
+  })
+
+  // Print each group to the terminal
+  mapObj(groups, (name, items) => {
+    Logger.yellow(`  ${ wordCaps(name) }`)
```

Comment:
Super minor and not at all blocking, but any chance we can make the titles of containers blue or green? lol

I associate yellow with warnings.

Reply:
Update to green